test/functional/simulator-e2e-specs.js

```diff
@@ -403,38 +311,28 @@ describe('advanced features', function () {
   describe(`setReduceMotion`, function () {
     it('should check accessibility reduce motion settings', async function () {
       await sim.setReduceMotion(true);
-      let fileSettings = await readSettings(sim, PLIST_IDENTIFIER.ACCESSIBLITY_SETTINGS);
-      for (const [, settings] of _.toPairs(fileSettings)) {
-        settings.ReduceMotionEnabled.should.eql(1);
-      }
-
       await sim.setReduceMotion(false);
-      fileSettings = await readSettings(sim, PLIST_IDENTIFIER.ACCESSIBLITY_SETTINGS);
-      for (const [, settings] of _.toPairs(fileSettings)) {
-        settings.ReduceMotionEnabled.should.eql(0);
-      }
     });
   });
 
   describe(`setReduceTransparency`, function () {
     it('should check accessibility reduce transparency settings', async function () {
       await sim.setReduceTransparency(true);
-      let fileSettings = await readSettings(sim, PLIST_IDENTIFIER.ACCESSIBLITY_SETTINGS);
-      for (const [, settings] of _.toPairs(fileSettings)) {
-        settings.EnhancedBackgroundContrastEnabled.should.eql(1);
-      }
-
       await sim.setReduceTransparency(false);
-      fileSettings = await readSettings(sim, PLIST_IDENTIFIER.ACCESSIBLITY_SETTINGS);
-      for (const [, settings] of _.toPairs(fileSettings)) {
-        settings.EnhancedBackgroundContrastEnabled.should.eql(0);
-      }
     });
   });
 
-  describe('updateSafariGlobalSettings', function () {
+  describe('Safari', function () {
+    it('should scrub Safari', async function () {
```

Comment:
hm, this test got failed. I re-ran the failed one but it got failed again

Reply:
yep, seems like the `ps` implementation has been outdated. Fixed now